src/events/EventResult.ts

```diff
@@ -0,0 +1,5 @@
+export enum EventResult {
```

Comment:
Maybe instead of resolving, use strings like other enums

@connerdouglass thoughts?

Reply:
Yes I think we should use strings in the enum. I'll also comment on `InventoryInteractEvent` with how we should update that.

Reply:
Changed to be consistent with the rest of the project